src/main/scala/codacy/docker/api/DockerApi.scala

```diff
@@ -50,3 +50,13 @@ object MetricsConfiguration {
 
   trait Value extends Any
 }
+
+case class DuplicationConfiguration(language: Option[Language],
```

Comment:
Can you run without the language?

Reply:
Not if you are running PMD CPD :/ . Will correct it, thanks